backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -585,18 +585,24 @@ def make_role(self, data: typing.Dict[str, typing.Any]) -> object:
 class WorkspaceMemberInviteSchema(marshmallow.Schema):
     role = marshmallow.fields.String(
         example='contributor',
-        validate=OneOf(UserRoleInWorkspace.get_all_role_slug())
+        validate=OneOf(UserRoleInWorkspace.get_all_role_slug()),
+        required=True
     )
     user_id = marshmallow.fields.Int(
         example=5,
         default=None,
         allow_none=True,
     )
-    user_email_or_public_name = marshmallow.fields.String(
+    user_email = marshmallow.fields.String(
```

Comment:
you must verify that email is an email `marshmallow.fields.Email`

Reply:
you're right.